recipes-wpe/wpebackend-rdk/wpebackend-rdk_0.1.bb

```diff
@@ -57,6 +57,8 @@ PACKAGECONFIG[libinput]         = "-DUSE_INPUT_LIBINPUT=ON,-DUSE_INPUT_LIBINPUT=
 PACKAGECONFIG[libinput-udev]    = "-DUSE_INPUT_UDEV=ON,-DUSE_INPUT_UDEV=OFF,libinput,"
 PACKAGECONFIG[virtualinput]     = "-DUSE_VIRTUAL_KEYBOARD=ON,-DUSE_VIRTUAL_KEYBOARD=OFF,wpeframework,"
 
+CXXFLAGS += "${@bb.utils.contains('DISTRO_FEATURES', 'wayland', '-DWL_EGL_PLATFORM', '', d)}"
```

Comment:
This applies to both the Westeros as well as the WPEFramework backends?

Reply:
yes, Irrespective of backends this will be applied. will not be applicable for wpe-eglfs-image as eglfs removes wayland distro